internal/wstrust/saml_token_info.go

```diff
@@ -5,6 +5,11 @@ package wstrust
 
 type SamlAssertionType int
 
+// TODO(msal reviewer): I love iota, but...  It is only safe if this value
+// is never sent to another service or written to disk for retrieval. Otherwise
+// the value should be statically assigned.  If someone can verify that.
+// Also, I couldn't figure out why not just use msalbase.SAMLV1Grant... here
+// instead of these values, instead we insert them in default_web_request_manager.go.
```

Comment:
I would not use the iota, based on some quick reading. using msalbase.SAMLV1Grant makes sense